docs/source/jit_unsupported.rst

```diff
@@ -0,0 +1,96 @@
+.. _jit_unsupported:
+
+TorchScript Unsupported Pytorch Constructs
+============================================
+
+Torch and Tensor Unsupported Attributes
+------------------------------------------
+
+
+TorchScript supports most methods defined on `torch` and `torch.Tensor`, but we do not have full coverage.
+Here are specific known ops and categories of ops which have diverging behavior between
+Python and TorchScript. Deprecated ops are not listed below.
```

Comment:
Why not list deprecated ops? We are detecting them below, we could also tag them with something  like `* resize (this method is deprecated and won't be supported)`

Reply:
Sure, we can do that